automation/deploy_hco.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -ex
+
+HCO_VERSION=1.4.0
+K_COMMAND=oc
+NS=kubevirt-hyperconverged
+
+# Create a catalog source
+cat <<EOF | ${K_COMMAND} apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-unstable-catalog-source
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: "quay.io/kubevirt/hyperconverged-cluster-index:${HCO_VERSION}"
+  displayName: Kubevirt Hyperconverged Cluster Operator
+  publisher: Kubevirt Project
+EOF
+
+# Create the kubevirt-hyperconverged namespace
+cat <<EOF | ${K_COMMAND} apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: "${NS}"
+EOF
+
+# Create operator group
+cat <<EOF | ${K_COMMAND} apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: kubevirt-hyperconverged-group
+    namespace: "${NS}"
+EOF
+
+# Create Subscription
+cat <<EOF | ${K_COMMAND} apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: hco-operatorhub
+    namespace: "${NS}"
+spec:
+    source: hco-unstable-catalog-source
+    sourceNamespace: openshift-marketplace
+    name: community-kubevirt-hyperconverged
+    channel: "${HCO_VERSION}"
+EOF
+
+# Wait for the deployments to be available
+"${K_COMMAND}" wait -n "${NS}" deployment hco-operator --for=condition=Available --timeout="1080s"
+"${K_COMMAND}" wait -n "${NS}"  deployment hco-webhook --for=condition=Available --timeout="1080s"
+
+# Create the HyperConverged resource
+curl "https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/main/deploy/hco.cr.yaml" | ${K_COMMAND} -n "${NS}" apply -f -
```

Comment:
you use HCO version 1.4.0, but here deploy the CR from master branch, not release-1.4

Reply:
I think it's a low risk. unknown fields are ignored while maintaining two files in two different places is also risky.